css/main.css

```diff
@@ -0,0 +1,75 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 700;
+}
+
+::-webkit-scrollbar {
+  width: 6px;
+  height: 10px;
+  display: none;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.logo > img {
+  width: 80px;
+  height: 80px;
+  padding: 10px;
+}
+
+header > .button {
+  padding: 30px;
+}
+
+.hero-section {
+  background-image: url("../image/heroImage.jpg");
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.title {
+  font-size: 30px;
+  margin: auto;
+  text-align: left;
+}
+
+.title > h1 {
+  margin-bottom: 0;
+}
+
+.description {
+  margin: auto;
+  font-size: 14px;
+  text-align: left;
+  font-weight: normal;
+}
+
+ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  justify-content: flex-start;
+  margin: 0;
+  padding: 0;
+}
+
+.icons > ul > li {
+  margin-left: 20px;
+  margin-top: 5px;
+}
+
+.icons > ul > li > a:hover { color: #7f8cff; }
+
+.social {
+  color: #7f8cff;
+}
+
+.section2 {
+  margin-top: 200px;
+}
```

Comment:
- [ ] Kyndly stick as much as you can to the figma design.

![image](https://user-images.githubusercontent.com/95243996/191729762-f40293ea-dff0-4ef0-8784-272ae9999a55.png)

![image](https://user-images.githubusercontent.com/95243996/191730208-ef11d0f9-ed1e-44c3-9e1e-0d83cf7c9a3b.png)

![image](https://user-images.githubusercontent.com/95243996/191730299-e805797a-c38e-4571-8244-bad1c6b8d3a0.png)